lib/Interpreter/DeclUnloader.cpp

```diff
@@ -649,7 +649,12 @@ bool DeclUnloader::VisitRedeclarable(clang::Redeclarable<T>* R, DeclContext* DC)
     GD = GlobalDecl(CXXDtor, Dtor_Base);
     MaybeRemoveDeclFromModule(GD);
     GD = GlobalDecl(CXXDtor, Dtor_Comdat);
+#if defined(LLVM_ON_WIN32)
+    // MicrosoftMangle.cpp:954 calls llvm_unreachable when mangling Dtor_Comdat
```

Comment:
Is that a current limitation of the MS mangler?

If this was something we shouldn't try to mangle, can't we skip such declarations as `Dtor_Comdat` in `MaybeRemoveDeclFromModule` instead?